src/java/org/apache/cassandra/db/Directories.java

```diff
@@ -1018,42 +1013,6 @@ protected static SnapshotManifest maybeLoadManifest(String keyspace, String tabl
         return null;
     }
 
-    public List<String> listEphemeralSnapshots()
-    {
-        final List<String> ephemeralSnapshots = new LinkedList<>();
-        for (File snapshot : listAllSnapshots())
-        {
-            if (getEphemeralSnapshotMarkerFile(snapshot).exists())
-                ephemeralSnapshots.add(snapshot.name());
-        }
-        return ephemeralSnapshots;
-    }
-
-    private List<File> listAllSnapshots()
```

Comment:
why is this removed?

Reply:
I think it is because that method is not necessary anymore. We are doing all listing of snapshots via `SnapshotLoader`.

Reply:
ok thanks for clarifying.